Server/BackEndMonitoringServer/include/CProcess.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <windows.h>
+#include <psapi.h>
+#include <functional>
+
+#include "EMemoryCountType.h"
+#include "EConvertValueFromBytes.h"
```

Comment:
Looks like this include not required here

Reply:
EMemoryCountType is required for m_count_type

Reply:
EConvertValueFromBytes - deleted, EMemoryCountType changed to forward declaration, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4#diff-0544ae2661f9747e70ea10c500758935c3f7d93fc11a7c69c55cadb18011b360)